### DISCUSSION TITLE
Has anyone been able to run on apple silicon?

### DISCUSSION THREAD

**Discussion by drewcapener** [openwallet-foundation/bifold-wallet#228] [Q&A]
Hi, I've just been familiarizing myself with some of the repos in the Aries project. I wanted to try and install and run the mobile agent but haven't been able to get it to run (trying for ios) on my apple silicon Mac by following the provided instructions. I am assuming my issues must be coming from the different architecture. Has anyone been able to get this running or does anyone have any thoughts as to how I might approach getting it running? I'm happy to provide further details (errors, etc.) if that would be helpful. Thanks in advance!

**Comment by realabbas** [comment]
> Hi, I've just been familiarizing myself with some of the repos in the Aries project. I wanted to try and install and run the mobile agent but haven't been able to get it to run (trying for ios) on my apple silicon Mac by following the provided instructions. I am assuming my issues must be coming from the different architecture. Has anyone been able to get this running or does anyone have any thoughts as to how I might approach getting it running? I'm happy to provide further details (errors, etc.) if that would be helpful. Thanks in advance!

Hi there, yes I'm able to run the app on Apple Mac Silicon M1 for Android version on emulator. The iOS emulator isn't yet supported by this project.

<img width="1440" alt="Screenshot 2022-04-21 at 2 05 46 PM" src="https://user-images.githubusercontent.com/43527087/164448303-339626d8-e6cc-425c-b1e4-b0ea107177cc.png">

**Comment by cvarjao** [comment]
iOS support for Emulator in M1 is now supported
